lib/node_modules/@stdlib/blas/ext/base/sdsnansum/docs/repl.txt

```diff
@@ -1,9 +1,11 @@
 
 {{alias}}( N, x, stride )
-    Computes the sum of single-precision floating-point strided array elements,
-    ignoring `NaN` values and using extended accumulation.
+    Computes the sum of single-precision
+    floating-point strided array elements,
+    ignore `NaN` values and using extended accumulation.
```

Comment:
This is not properly wrapped at 80 chars. See https://github.com/stdlib-js/stdlib/blob/develop/docs/repl_text.md#wrapping

Reply:
Have fixed